### ISSUE TITLE
sign in page is sending "GET" request instead of "POST" request

### ISSUE THREAD

**Issue by chirayu-humar** [MonalikaPatnaik/UMatter#174]
i observed that the sign in page is sending GET request which is use less for functional requirements like Sign in. GET request do't have body hence the data intered by user will be sent to the server using path parameters or query parameters which is heighly unsecure way to send data to server. i can make the sign in process more secure as well as proper in will send all the informations in encryped format using an npm packege (bcrypt). the sensative data (like password and email) will be send to server in the form of JSON object to the server and in body of post request and hence will not be visible in URL and that too in encrypted form. 

in this way the whole sign in process will be heighly secure. i will implement this to sign up page also because both sign in and sign up pages will have to access same data base. i will integrate "sqlite data base" using npm pakeges like "sqlite" and 
"sqlite3" hence data entered will be saved in server that too in encrypted format and can be verified at the time of sign in process. 

when the server will receave sign in request server will generate a personal access token to user in body of response through which user will not have to enter password again untill and unless the token is expired. 

basically i wanted to conclude with last words which are, i will transform login and sign up system of this website which is nearly nothing in site currently. 

i am attaching the screen short of sign in attempt which i have made and screen shot of header of request made which is proving that request make is a GET request. this login and sign up system is at a very nascent stage, needs to improve alot in which i can realy support organisation. 

![Screenshot from 2023-05-21 17-45-07](https://github.com/MonalikaPatnaik/UMatter/assets/97947258/80b5a2ac-80d8-4393-a4b5-6015d23cfed9)
![Screenshot from 2023-05-21 17-47-37](https://github.com/MonalikaPatnaik/UMatter/assets/97947258/4e37da84-596b-43f3-aa52-4806228a0969)

**Comment by MonalikaPatnaik** [comment]
Hey..sure..you can start working on this!
Thanks for contributing!

**Comment by chirayu-humar** [comment]
thanks for assigning me this issue, i am now starting working on it

**Comment by chirayu-humar** [comment]
please allow me to work for one more day, all main functionalities are done, just working on giving final touch, will genereate a PR before today's night or tomarrow morning.

**Comment by chirayu-humar** [comment]
i have sent one pull request, please review that, waiting for response, thanks

**Comment by MonalikaPatnaik** [comment]
Hey @chirayu-humar..start working on this and submit a PR soon!
Thanks!
